### DISCUSSION TITLE
Invalid client_id error

### DISCUSSION THREAD

**Discussion by jainsrbh** [dexidp/dex#2675] [Q&A]
Not Found
Invalid client_id ("694969472394.apps.googleusercontent.com").

Logs:
Sep 21, 2022 @ 14:20:21.372 | Failed to parse authorization request: Invalid client_id ("694969472394.apps.googleusercontent.com").
-- | --

I am hitting above error post restarting dex pod in Kubernetes and changing issuer URL in the configuration file. 

Configuration file:
`issuer: https://sso-dev.io
storage:
    type: kubernetes
    config:
        inCluster: true
logger:
    level: debug
    format: json
expiry:
    deviceRequests: 5m
    signingKeys: 6h
    idTokens: 24h
    refreshTokens:
        disableRotation: false
        reuseInterval: 3s
        validIfNotUsedFor: 2160h
        absoluteLifetime: 3960h
oauth2:
    responseTypes:
        - code
    skipApprovalScreen: true
    alwaysShowLoginScreen: false
connectors:
    - type: google
      id: google
      name: Google
      config:
        clientID: 694969472394.apps.googleusercontent.com
        clientSecret: XX
        redirectURI: https://sso-dev.io/callback
        serviceAccountFilePath: /tmp/oidc/googleAuth.json
        adminEmail: XX
staticClients:
    - id: argocd
      redirectURIs:
        - https://argocd.infra.dev.io/auth/callback
        - https://argocd-dev.io/auth/callback
      name: ArgoCD
      secret: XX
enablePasswordDB: false
`
Kindly help please.

**Comment by yashg-op** [comment]
hi were you able to solve this?
